priv/perf/config/config.exs

```diff
@@ -5,13 +5,30 @@ use Mix.Config
 # https://github.com/googleapis/elixir-google-api/issues/26#issuecomment-360209019
 config :tesla, adapter: Tesla.Adapter.Hackney
 
+config :ethereumex,
+  url: System.get_env("ETHEREUM_RPC_URL")
+
 config :load_test,
   pool_size: 5000,
   max_connection: 5000,
-  faucet_default_funds: trunc(:math.pow(10, 18) * 50),
-  initial_funds_wei: trunc(:math.pow(10, 17)),
+  child_chain_url: System.get_env("CHILD_CHAIN_URL"),
+  watcher_security_url: System.get_env("WATCHER_SECURITY_URL"),
+  watcher_info_url: System.get_env("WATCHER_INFO_URL"),
+  faucet_private_key: System.get_env("LOAD_TEST_FAUCET_PRIVATE_KEY"),
+  eth_vault_address: System.get_env("CONTRACT_ADDRESS_ETH_VAULT"),
+  faucet_deposit_wei: trunc(:math.pow(10, 18)),
```

Comment:
This is pretty expensive 😢 and from experience we don't need this much.
10^12 ~10^15 have been quite enough for me

Reply:
Agreed. 
I increased it because the stress env currently has fees set to 30_000_000_000_000,
but I'll change the default and override in stress config.
(or even better, change the stress env fees)